packages/core/src/explorers/starkCompass.ts

```diff
@@ -0,0 +1,33 @@
+import { Chain } from "@starknet-react/chains";
+import { Explorer, ExplorerFactory } from "./explorer";
+
+// Define the StarkcompassExplorer class that extends Explorer
+export class StarkcompassExplorer implements Explorer {
```

Comment:
```suggestion
// Define the StarkCompassExplorer class that extends Explorer
export class StarkCompassExplorer implements Explorer {
```

Rename for consistency.